styles.css

```diff
@@ -179,36 +286,59 @@ article:hover p, article:focus-within p {
 }
 
 @media screen and (max-width: 630px) {
+	html {
+		font-size: 18px;
```

Comment:
There's probs a real reason for this but you've used `rem` the whole way through  which is meant to help with responsiveness but then used px in the media query?

Reply:
The rems are responding to the html absolute font-size I believe